src/components/forms/ProfileUpdateForm.js

```diff
@@ -135,13 +134,13 @@ const componentWithStyles = withStyles(styles)(ProfileUpdateForm);
 
 function mapStateToProps(state) {
```

Comment:
short hand is probably cleaner

function mapStateToProps = (state) => ({ profile: state.profile });